core/config/configs.md

```diff
@@ -116,35 +116,26 @@ hand, we do know that there is a genuine need to have slightly
 different purposes expressed.
 
 This is why we've chosen to use the [Trait
-pattern](https://en.wikipedia.org/wiki/Trait_(computer_programming)),
+pattern](https://en.wikipedia.org/wiki/Trait_(computer_programming),
```

Comment:
There's an open paren for the link "[...](...)", and another for the wikipedia name. The original was correct.

Reply:
Interesting. You are correct. However, I was editing via Atom's Markdown editor with Markdown preview, which got confused by the parens within the URL. Changed to uuencode the parens, which seems to work both places.